.migration-backup/ui-components/src/services/__tests__/classService.security.test.ts

```
@@ -0,0 +1,716 @@
+    return {
+      eq: jest.fn(() => createMockChain()),
+      not: jest.fn(() => createMockChain()),
+      order: jest.fn(() => createMockChain()),
+      range: jest.fn(() => createMockChain()),
+      ilike: jest.fn(() => createMockChain()),
+      or: jest.fn(() => createMockChain()),
+      single: jest.fn(() => Promise.resolve({ data: { id: 1, school_id: 1 }, error: null })),
+      catch: jest.fn(() => Promise.resolve(mockResult)),
+      // Terminating operations that return promises
+      execute: jest.fn(() => Promise.resolve(mockResult)),
+      then: undefined // Explicitly remove thenable behavior
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)